### ISSUE TITLE
Invalid BTC address in addresses sanctioned by OFAC

### ISSUE THREAD

**Issue by 0xB10C** [0xB10C/ofac-sanctioned-digital-currency-addresses#5]
The following address was added in https://ofac.treasury.gov/recent-actions/20231003. It's not a valid Bitcoin address. 

https://github.com/0xB10C/ofac-sanctioned-digital-currency-addresses/blob/bff68619f9d4a417095189ca7403dc4136aa24b5/sanctioned_addresses_XBT.txt#L346

How should the tool handle this address? 
- One option is to hardcode it in the tool to not add it to the generated lists
- Another is to just leave it in and let downstream users take care of it (i.e. like https://github.com/0xB10C/miningpool-observer/pull/73)

**Comment by derekm** [comment]
Can you report the bug upstream to the SDN list admins?

FWIW, in my use case, I decode the addresses to retrieve the byte array of the hash that the address represents, so I will notice invalid addresses as invalid.

**Comment by 0xB10C** [comment]
> Can you report the bug upstream to the SDN list admins?

No, that's currently not one of my priorities - but if someone else wants to pick that up: feel free to do so :)
